math/minuit2/inc/Minuit2/GradientCalculator.h

```diff
@@ -35,6 +35,15 @@ class GradientCalculator {
 
    virtual bool G2(const MinimumParameters &, MnAlgebraicVector &) const { return false;}
 
+   /**
+    * Enable parallelization of gradient calculation using OpenMP.
+    * \param doParallel true to enable, false to disable
+    * \note If OPENMP is not available, this function has no effect
+    */
+   static void DoParallel(bool doParallel = true) { fDoParallel = doParallel; }
```

Comment:
Should this indicate in the signature that it is OMP/process as opposed to the default parallel mechanism elsewhere (i.e. usually threads/TBB)?